.env.example

```
@@ -29,3 +29,14 @@ DATABASE_USERNAME=postgres
 DATABASE_PASSWORD=postgres
 
 ################################################################################
+
+# Starknet
+STARKNET_RPC_URL=https://starknet-mainnet.public.blastapi.io/rpc/v0_8
+CAMPAIGN_CONTRACT_ADDRESS=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CAMPAIGN_CONTRACT_ADDRESS key should go before the STARKNET_RPC_URL key

(UnorderedKey)